docs/ddl-of-carbondata.md

```diff
@@ -608,12 +607,10 @@ CarbonData DDL statements are documented here,which includes:
   This can be SDK output or C++ SDK output. Refer [SDK Guide](./sdk-guide.md) and [C++ SDK Guide](./csdk-guide.md). 
 
   **Note:**
-  1. Dropping of the external table should not delete the files present in the location.
+  1. Dropping of the external table will not delete the files present in the location.
   2. When external table is created on non-transactional table data, 
     external table will be registered with the schema of carbondata files.
-    If multiple files with different schema is present, exception will be thrown.
-    So, If table registered with one schema and files are of different schema, 
-    suggest to drop the external table and create again to register table with new schema.  
+    If multiple files with different schema is present, exception will be thrown.  
```

Comment:
actually if multiple files different schema present, we check if same column present with different schema, in that case we throw exception, so here we can be more specific

Reply:
if column is not present then also we throw exception, right?

Reply:
it basically checks latest file and give data based on columns present in that file.

Reply:
done